### DISCUSSION TITLE
丢包率异常

### DISCUSSION THREAD

**Discussion by qw1642190428** [XIU2/CloudflareSpeedTest#517] [提问]
### 问题描述

我不知道怎么回事 优选出来的ip 丢包率 四个包 一直稳定在两个到三个之间 不管是反代还是官方

### 软件版本

2.2.5

### 附加截图

![image](https://github.com/XIU2/CloudflareSpeedTest/assets/54324781/94dd0faa-b458-4f94-8640-cb0681083aaa)

**Comment by XIU2** [comment]
你去[这里](https://www.elifulkerson.com/projects/tcping.php)下个 TCPing 工具来手动 TCPing 测试这些 IP（随便挑几个）试试看是否丢包。

**Reply by qw1642190428** [reply]
![image](https://github.com/XIU2/CloudflareSpeedTest/assets/54324781/d248f818-2bd9-474f-9aee-e8405cb8de3e)
![image](https://github.com/XIU2/CloudflareSpeedTest/assets/54324781/b7423a4f-712a-4e1d-9aaf-526ff822df4e)
试了两个ip tcping一个没有丢包

**Reply by qw1642190428** [reply]
这是我今天重新优选的ip

**Reply by qw1642190428** [reply]
应该不是系统问题 我以前也这样 后来硬盘坏了 换硬盘重装系统 还是这样 网络是秦皇岛联通

**Reply by XIU2** [reply]
你截图里测试的是 80 端口（要在 IP 后面加空格及端口，例如 tcping 1.2.3.4 443 这样），而 CloudflareST 截图里测试的是 443 端口（默认值）

另外，你尝试用 CloudflareST 只测速你截图里的那几个少量 IP 试试（用于验证是否因为测速 IP 多并发高导致的问题）。

**Reply by qw1642190428** [reply]
> 你截图里测试的是 80 端口（要在 IP 后面加空格及端口，例如 tcping 1.2.3.4 443 这样），而 CloudflareST 截图里测试的是 443 端口（默认值）
> 
> 另外，你尝试用 CloudflareST 只测速你截图里的那几个少量 IP 试试（用于验证是否因为测速 IP 多并发高导致的问题）。

我又用软件默认重新优选了一遍 十个ip 都丢了两个包 然后用排名前五的ip用tcpin 443端口 全通 不丢包 延迟不过300 单独用软件测 不丢包  那这应该是bug？

**Reply by XIU2** [reply]
你用 CloudflareST 只延迟测速**几个 IP** 试试（加上 -dd 禁用下载测速）。

同时你在找几个和 Cloudflare CDN 无关的 IP 延迟测速试试。

**Reply by XIU2** [reply]
对了，你再尝试手机打开流量数据关闭WIFI，然后用 USB 数据线共享给电脑上（电脑原先网线暂时先拔掉），然后再用 CloudflareST 延迟测速（记得加上 `-dd` 禁用下载测速，否则很耗流量）看看结果。

因为你遇到的问题比较奇怪，也没有其他人反馈过，我也不清楚具体原因，因此也只能依靠 排除法 一点一点排查了。

上面说的让 CloudflareST 延迟测速走手机流量，就是为了排查问题是不是和你电脑所用的宽带网络有关。
如果测速结果还是和以前一样，说明和宽带网络关系不大，如果测速结果正常了，显然就说明是宽带网络的问题了，才能进一步排查。

**Reply by qw1642190428** [reply]
搞了 手机网络测试淘宝 B站 百度的ip全通 不丢包 cf 还是那个样子 我不知道这三个有没有参考价值 我试了一下我的谷歌vps的ip 是不通的 然后用dd的电脑网络测试还是那样

**Reply by qw1642190428** [reply]
看了眼证书 都不是cf颁发的 应该没问题

**Reply by qw1642190428** [reply]
看样子可以排除宽带和电脑的问题了

**Reply by qw1642190428** [reply]
我用电脑和手机测的是原版的ip段 但是+dd 还是那样 不过貌似只要和cf沾边的ip 哪怕是反代 貌似都这样 我这里反代指的是电报里发布的那些扫描ip

**Reply by XIU2** [reply]
你的描述有点乱，我也看不太懂。
你就直接说：手机共享网络下，CloudflareST 测速哪些 IP 结果是否正常，且最好是国外 IP（随便找一些国外网站的解析 IP）。

****

也就是该问题只局限于和 Cloudflare CDN 相关的 IP？其他 IP 都正常？

鉴于没什么其他人出现该问题，初步猜测可能是你这边网络与 Cloudflare CDN 之间的问题？

说实话，这种问题我一般都归为疑难杂症，我个人也完全没有任何头绪，想不到什么原因会导致这种奇怪的问题。。。

**Reply by qw1642190428** [reply]
![image](https://github.com/XIU2/CloudflareSpeedTest/assets/54324781/9c79aca3-bf04-4c39-bc96-1b652bdf5995)
我用手机共享测的几个中国没有被墙的国外网站ip

**Reply by qw1642190428** [reply]
这五个ip是这些网站的
![image](https://github.com/XIU2/CloudflareSpeedTest/assets/54324781/a4f2a1b3-da72-498b-8a46-516ad9398e2b)

**Reply by XIU2** [reply]
那就是我前面说的：

> 也就是该问题只局限于和 Cloudflare CDN 相关的 IP？其他 IP 都正常？

> 鉴于没什么其他人出现该问题，初步猜测可能是你这边网络与 Cloudflare CDN 之间的问题？

> 说实话，这种问题我一般都归为疑难杂症，我个人也完全没有任何头绪，想不到什么原因会导致这种奇怪的问题。。。

**Reply by qw1642190428** [reply]
可能吧 有一段时间是正常的 时而好时而坏
